solr/modules/analytics/src/test/org/apache/solr/analytics/legacy/facet/LegacyFieldFacetCloudTest.java

```diff
@@ -442,48 +440,48 @@ public void sumTest() throws Exception {
     String responseStr = response.toString();
 
     // Int Date
-    Collection<Double> intDate =
+    ArrayList<Double> intDate =
```

Comment:
Probably, but the method actually returns an ArrayList itself. I didn't change the method response. The reason it can't be a collection is that is ambiguous when comparing assertEquals - errorprone catches this.